variants/heltec_esp32c3/variant.h

```diff
@@ -1,16 +1,43 @@
-#define BUTTON_PIN 9
-
-// LED pin on HT-DEV-ESP_V2 and HT-DEV-ESP_V3
-// https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
-// https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
-#define LED_PIN 2 // LED
-#define LED_INVERTED 0
+/*
+ * Heltec HT-CT62, Heltec HT-DEV-ESP V2 and Heltec HT-DEV-ESP V3
+ * Heltec HT-CT62 Schematic: https://resource.heltec.cn/download/HT-CT62/HT-CT62_Reference_Design.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V2_Sch.pdf
+ * Heltec HT-DEV-ESP V3 Schematic: https://resource.heltec.cn/download/HT-DEV-ESP/HT-DEV-ESP_V3_Sch.pdf
+ */
 
 #define HAS_SCREEN 0
+
+// Board does not have enough free pins for a UART GPS, use an I2C GPS
 #define HAS_GPS 0
```

Comment:
I2C GPS is not supported on Meshtastic.

Reply:
Thanks for that. I had kinda come to work that out yesterday but just wish i had dug a little deeper before spending a lot of time mucking around with it. A man with a little knowledge is a man with no knowledge at all.

Reply:
> I2C GPS is not supported on Meshtastic.

Fair enough, let me remove the remark about GPS then. The limited I/O is really getting tough to make decisions with...

GPIO18 = I2C_SCL (could have been USB_DN or UART1)
GPIO19 = I2C_SDA (could have been USB_DP or UART1)
GPIO20 = U0RXD
GPIO21 = U0TXD

I feel like this has the most utility for most "classic" MCU hackers who are used to getting to their boards over TTL serial and want to have I2C peripherals.

Maybe U0RXD and U0TXD can be sacrificed for GPS too, making the port do double duty as flashing UART and GPS UART?

Not really sure what's preferred/most useful at this point. So I guess this PR is becoming a bit of an RFC.

Reply:
> Thanks for that. I had kinda come to work that out yesterday but just wish i had dug a little deeper before spending a lot of time mucking around with it. A man with a little knowledge is a man with no knowledge at all.

There is a way to get both I2C and GPS UART working, which is to sacrifice UART0 entirely and reassign the pins to GPS.

That means that you have to disconnect the GPS while putting the ESP32 in download mode, flash your firmware, then reconnect the GPS. There isn't even 1 more pin of GPIO free to control the enable pin of the GPS.

Alternatively, we can use GPIO0 or GPIO1 to drive the GPS enable, but then HT-DEV-ESP board users have to desolder the 32kHz crystal to free up GPIO0 and GPIO1.